src/query/scan_manager.c

```diff
@@ -7987,7 +8056,7 @@ scan_hash_probe_next (THREAD_ENTRY * thread_p, SCAN_ID * scan_id, QFILE_TUPLE *
 
 /*
  * check_hash_list_scan () - Check if hash list scan is possible
- *   return: bool
+ *   return: int  1: in-memory 2: hybrid in-memory
```

Comment:
trivial; Defining it with enum type would better for readability

Reply:
okay. I'll fix it